src/ManifoldsBase.jl

```diff
@@ -511,41 +575,53 @@ end
 @doc doc"""
     injectivity_radius(M::Manifold, x)
 
-Distance $d$ such that `exp(M, x, v)` is injective for all tangent
-vectors shorter than $d$ (has a left inverse).
+Distance $d$ such that [`exp(M, x, v)`](@ref exp(::Manifold, ::Any, ::Any)) is
+injective for all tangent vectors shorter than $d$ (i.e. has a left inverse).
 """
-injectivity_radius(M::Manifold, x) = Inf
+injectivity_radius(M::Manifold, x) = injectivity_radius(M)
 
 @doc doc"""
     injectivity_radius(M::Manifold, x, R::AbstractRetractionMethod)
 
-Distance $d$ such that `retract(M, x, v, R)` is injective for all tangent
-vectors shorter than $d$ (has a left inverse).
+Distance $d$ such that
+[`retract(M, x, v, R)`](@ref retract(::Manifold, ::Any, ::Any, ::AbstractRetractionMethod))
+is injective for all tangent vectors shorter than $d$ (i.e. has a left inverse).
 """
 injectivity_radius(M::Manifold, x, ::AbstractRetractionMethod) = injectivity_radius(M, x)
 
 """
     injectivity_radius(M::Manifold)
 
-Infimum of the injectivity radii of all manifold points.
+Infimum of the [`injectivity_radius`](@ref injectivity_radius(::Manifold, ::Any))
+of all manifold points.
 """
 injectivity_radius(M::Manifold) = Inf
```

Comment:
If the above local version defaults to this conservative choice, this should not be `Inf` but return an error, that its not implemented. `Inf` is far too large (for the `Sphere` for example).

Reply:
Good point, I'll change.

Reply:
Technically, that's a breaking change, so per semver we'll need to bump the version number to 0.3.

Reply:
Yes, well. But it's important, I think, since `Inf` might really have unwanted side effects.

Reply:
Agreed